extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -269,6 +269,7 @@ oppia.directive('graphViz', function() {
               y: $scope.state.mouseY,
               label: ''
             });
+            setMode0(_MODES.MOVE);
```

Comment:
Yeah in my opinion "onClickModeButton" and "setMode" would be better corresponding to their functionality